### PR TITLE
Added logo, favicon, and fixed path for examples in toc.yml

### PR DESCRIPTION
**Pull request by flinde**
Added logo, favicon, and fixed path for examples in toc.yml för docfx documentation project.

### CODE REVIEW

File: docfx_project/docfx.json [hunk 1]
```diff
@@ -59,7 +61,8 @@
     "globalMetadataFiles": [],
     "fileMetadataFiles": [],
     "template": [
-      "default"
+      "default",
+	  "./template/playwright-sharp"
```
Comment (kblok) [inline docfx_project/docfx.json:65]: Do we need this?

**Comment by kblok** [comment]
Can we also change the "D" logo on the header?
image

File: docfx_project/docfx.json (hunk 1 above)
Comment (flinde) [reply docfx_project/docfx.json]: No, I have removed the refrence to "./template/playwright-sharp"

**Comment by flinde** [comment]
I'm using docfx 2.56.2.0 and when I'm run >docfx docfx_project\docfx.json --serve I get the SVG with the 🎭emoji embeded as text.  Could it be a browser cache issue or is it different version of docfx maybe?

File: docfx_project/logo.svg [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+     inkscape:zoom="10.618421"
+     inkscape:cx="25.333333"
+     inkscape:cy="11.528946"
+     inkscape:window-x="0"
+     inkscape:window-y="0"
+     inkscape:window-maximized="0"
+     inkscape:current-layer="svg3968" />
+  <text
+     id="text3966"
+     x="-18.57333"
```
Comment (kblok) [inline docfx_project/logo.svg:53]: What do you think?

```suggestion
     x="0"
```

Before
image

After
image

**Pull request merged**